freeipa/src/main/java/com/sequenceiq/freeipa/converter/backup/BackupConverter.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.freeipa.converter.backup;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.common.api.cloudstorage.old.AdlsGen2CloudStorageV1Parameters;
+import com.sequenceiq.common.api.cloudstorage.old.S3CloudStorageV1Parameters;
+import com.sequenceiq.common.api.telemetry.request.LoggingRequest;
+import com.sequenceiq.common.api.telemetry.request.TelemetryRequest;
+import com.sequenceiq.freeipa.api.model.Backup;
+import com.sequenceiq.freeipa.configuration.BackupConfiguration;
+
+@Component
+public class BackupConverter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(BackupConverter.class);
+
+    private final boolean freeIpaBackupEnabled;
+
+    private final BackupConfiguration backupConfiguration;
+
+    public BackupConverter(BackupConfiguration backupConfiguration,
+            @Value("${freeipa.backup.enabled:true}") boolean freeIpaBackupEnabled) {
+        this.backupConfiguration = backupConfiguration;
+        this.freeIpaBackupEnabled = freeIpaBackupEnabled;
+    }
+
+    public Backup convert(TelemetryRequest request) {
```

Comment:
it's a bit strange that we use the telemetry settings for backup. Is it really a good idea?

Reply:
This is a shorter-term decision so we did not have to introduce using another s3 bucket or azure container.  We do not have time to introduce all the UI and flow components to support the additional storage location at the moment.  The Logging bucket/container is currently the only one available to the freeipa at the moment.

Reply:
i would suggest to support this from the API (env or probably better for freeipa) in this PR already, as backup is nothing to do with telemetry, especially with logging. use this "temporal" solution (to fill data from telemetry) only if backup request is not filled in the freeipa request. so if the UI will support it you can remove the logging usage (but you will have everything that is needed for the UI to implement that)

Reply:
We can change this when we add support for UI configuration.  I've added https://jira.cloudera.com/browse/CB-6146